### PR TITLE
Fix layout evaluation 

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

File: app/views/admin/evaluations/_form.html.erb [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <%= form_for @evaluation, url: admin_user_evaluations_path do |f| %>
   <div class="row">
     <div class="col-md-8">
-      <%= @user.name %>
-    </div>
-    <div class="col-md-4">
+      <%= @user.name %><br>
       <%= f.submit (f.object.new_record? ? t("evaluations.create")
         : t("evaluations.update")), class: "btn btn-" %>
```
Comment (yongsokheng) [inline app/views/admin/evaluations/_form.html.erb]: btn-? la gi?

File: app/assets/stylesheets/admin/evaluation.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#evaluations {
+  .tbl-evaluation {
+    .name {
+      width: 80%;
+    }
+    .action {
+      width: 10%;
```
Comment (thiennv57) [inline app/assets/stylesheets/admin/evaluation.scss]: 20%

**Comment by yongsokheng** [comment]
OK

**Comment by thiennv57** [comment]
OK

**Pull request merged**